<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmUpdate</class>
 <widget class="QWidget" name="frmUpdate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource/resources.qrc">
    <normaloff>:/images/iconfinder__snowflake_1679761_ico.ico</normaloff>:/images/iconfinder__snowflake_1679761_ico.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Buscar atualizações</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="textBrowser"/>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource/resources.qrc"/>
  <include location="../resource/resources.qrc"/>
 </resources>
 <connections/>
</ui>
